appinventor/appengine/src/com/google/appinventor/client/Ode.java

```diff
@@ -422,14 +423,14 @@ public void run() {
    */
 
   public void switchToTrash() {
-    Ode.getInstance().getTopToolbar().updateMoveToTrash("Delete From Trash");
+    getInstance().getTopToolbar().updateMoveToTrash("Delete From Trash");
```

Comment:
Since this is already in a method, we shouldn't need to call `getInstance()` since the implied `this` is the instance.